### PR TITLE
CHANGELOG 5.40.0

### PR DESCRIPTION
**Pull request by SarahFrench**
Partner Beta PR is here: https://github.com/hashicorp/terraform-provider-google-beta/pull/7859

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+BUG FIXES:
+* bigquery: fixed a permadiff when handling "assets" in `params` in the `google_bigquery_data_transfer_config` resource ([#18898](https://github.com/hashicorp/terraform-provider-google/pull/18898))
+* bigquery: fixed an issue preventing certain keys in `params` from being assigned values in `google_bigquery_data_transfer_config` ([#18888](https://github.com/hashicorp/terraform-provider-google/pull/18888))
+* compute: fixed perma-diff of `advertised_ip_ranges` field in `google_compute_router` resource ([#18869](https://github.com/hashicorp/terraform-provider-google/pull/18869))
+* container: fixed perma-diff on `node_config.guest_accelerator.gpu_driver_installation_config` field in GKE 1.30+ in `google_container_node_pool` resource ([#18835](https://github.com/hashicorp/terraform-provider-google/pull/18835))
+
+## v5.39.1 (July 30th, 2024)
+
+BUG FIXES:
+* datastream: fixed a breaking change in 5.39.0 `google_datastream_stream` that made one of `destination_config.bigquery_destination_config.merge` or `destination_config.bigquery_destination_config.append_only` required ([#18903](https://github.com/hashicorp/terraform-provider-google/pull/18903))
```
Comment (SarahFrench) [inline CHANGELOG.md:26]: This is currently present on main but wasn't present in the commit the release was cut from

**Review by c2thorn — APPROVED** [review]
LGTM

**Review by c2thorn — CHANGES_REQUESTED** [review]
requesting https://github.com/hashicorp/terraform-provider-google/pull/18962

**Comment by SarahFrench** [comment]
I've cherry-picked https://github.com/hashicorp/terraform-provider-google/commit/96f91a5c807e275ceedc167c43166ea3026f0bc5 into the release branch, updating CHANGELOG next

**Comment by SarahFrench** [comment]
Test builds:

[GA Cloud SQL](https://hashicorp.teamcity.com/buildConfiguration/TerraformProviders_GoogleCloud_GOOGLE_NIGHTLYTESTS_GOOGLE_PACKAGE_SQL/208797?hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandBuildChangesSection=true)
[GA BigQuery Connection](https://hashicorp.teamcity.com/buildConfiguration/TerraformProviders_GoogleCloud_GOOGLE_NIGHTLYTESTS_GOOGLE_PACKAGE_BIGQUERYCONNECTION/208798?hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandBuildChangesSection=true)
[GA Database Migration](https://hashicorp.teamcity.com/buildConfiguration/TerraformProviders_GoogleCloud_GOOGLE_NIGHTLYTESTS_GOOGLE_PACKAGE_DATABASEMIGRATIONSERVICE/208799?hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandBuildChangesSection=true)

**Review by c2thorn — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (melinath) [reply CHANGELOG.md:27]: has the fix been cherry-picked into the release branch?

Comment (melinath) [reply CHANGELOG.md:27]: oh I see - you mean the log. I see this commit in the history 👍

**Review by melinath — APPROVED** [review]

**Pull request merged**